### DISCUSSION TITLE
The mouse mode does not work well on the Android version.

### DISCUSSION THREAD

**Discussion by zhangbo8418** [rustdesk/rustdesk#5094] [General]
### Bug Description

The mouse mode does not work well on the Android version.

### How to Reproduce

When the Android device is connected to a Mac or Windows computer, dragging the mouse in mouse mode.

### Expected Behavior

I hope to be able to use the mouse mode properly.

### Operating system(s) on local side and remote side

Android -> All

### RustDesk Version(s) on local side and remote side

1.2x

### Screenshots

https://github.com/rustdesk/rustdesk/assets/46797568/3c88e150-261c-4e75-b408-c5810afa297a

**Comment by rustdesk** [comment]
I guess I know what you will say, but ... :)

https://rustdesk.com/docs/en/manual/mobile/

<img width="227" alt="image" src="https://github.com/rustdesk/rustdesk/assets/71636191/bf78a131-aef0-429b-8c06-90ea0a57e717">

**Reply by zhangbo8418** [reply]
How can I move my mouse cursor? When I move my mouse now, it's like the left mouse button is being held down...

**Reply by rustdesk** [reply]
After releasing finger after double tap & move, it returns to normal mouse moving.  This is what I see on my android.

**Reply by zhangbo8418** [reply]
Normally, moving the mouse cursor after double-clicking is dragging, but now moving the mouse is dragging, like a mouse with a broken left button.

**Reply by rustdesk** [reply]
> I guess I know what you will say, but ... :)

**Reply by zhangbo8418** [reply]
I didn't double click. Just like the left button of the mouse is broken, the left button is always pressed, and I can't release it after double-clicking, and it's the same when I move the cursor again. . .

**Reply by zhangbo8418** [reply]
You should be able to see my touch operation in the video, I didn't double-tap. . .

**Comment by rustdesk** [comment]
nightly build?

**Reply by rustdesk** [reply]
@fufesou your latest commit about mouse broke this.

**Reply by zhangbo8418** [reply]
yes

**Reply by rustdesk** [reply]
So please specify clearly next time.

**Reply by rustdesk** [reply]
:(

**Reply by zhangbo8418** [reply]
Sorry, I thought my video made this bug very clear. . .

**Reply by rustdesk** [reply]
Your video is clear, but I tested with 1.2.1 release (because you specified 1.2.x). I did not believe what I saw in your video. I chose to believe my test. :(
Eventually I downloaded nightly build and test, I got the issue.

**Comment by rustdesk** [comment]
> After releasing finger after double tap & move, it returns to normal mouse moving. This is what I see on my android.

@fufesou  not return to normal.

**Comment by rustdesk** [comment]
nightly build updated for this. Please test.

**Reply by GHGiampy** [reply]
I’m not sure if this is a new regression, more than a year and a half later, or I'm simply unlucky, but I’m facing the same problem with the release version 1.3.3.
I'm using Android connected to Windows 10, if it matters.

**Reply by rustdesk** [reply]
fixed in 1.3.4/1.3.5

**Reply by GHGiampy** [reply]
Tested version 1.3.5, my bad.
Mouse mode works perfectly, but I’m experiencing the issue mentioned by the OP in touch mode. Specifically, as soon as I drag a finger (without double-clicking), the selector is triggered as if the left mouse button were pressed.
Should I open a new thread/issue?

**Reply by rustdesk** [reply]
that is how touch mode work, guy

# i know what you will say then.   

but please do not @ me

**Reply by zhangbo8418** [reply]
There's nothing wrong with it, it's all normal now. Are there any plans to support mouse mode on Android hosts?

**Reply by GHGiampy** [reply]
Oh ok, sorry for the misunderstanding.
Thank you so much.
